Userland/Libraries/LibWeb/Page/Frame.h

```diff
@@ -41,6 +41,8 @@
 
 namespace Web {
 
+static constexpr size_t max_recursion_depth = 3;
```

Comment:
This name is way too generic to put in the `Web` namespace.
Surely there are a bunch of other things that may also want to have a recursion limit. :)
Since it's not actually used outside of `is_frame_nesting_allowed()`, let's just put it in there.